WeakAuras/Types.lua

```diff
@@ -2870,9 +2870,14 @@ do
     [174] = L["Heroic Party"],
     [175] = L["10 Player Raid"],
     [176] = L["25 Player Raid"],
+    [184] = L["Normal Party"],
+    [185] = L["20 Player Raid"],
+    [186] = L["40 Player Raid"],
     [192] = L["Dungeon (Mythic+)"], -- "Challenge Level 1" TODO: check if this label is correct
     [193] = L["10 Player Raid (Heroic)"],
     [194] = L["25 Player Raid (Heroic)"],
+    [197] = L["10 Player Raid"],
+    [198] = L["Normal Party"],
```

Comment:
i would love to be a fly on the wall when these ids are chosen. 185 (20 man raid), 186 (40 man raid), but then 1***9***7 (10 man raid)??

Reply:
Hah yeah! When in doubt choose a random number.